index.html

```diff
@@ -305,13 +305,33 @@ <h2>Languages</h2>
                         </div>
 
                     </div>
+            </section>
 
+            <!--Added by Sandeep (mailsg0912@gmail.com)-->
 
-                   </section>
-               <section  id="contact"></section>
-
-            </div>
-
+            <section id="contact">
+                <div class="contact-shape">
+                </div>
+                <div class="contact-form">
+                    <h2>Contact me</h2>
+                    <div class="contact-description">
+                        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                            I’d love to help with it!
+                        </p>
+
+                        <form action="#" method="">
```

Comment:
- Hey, let's ensure we can collect the form data by using the [Formspree service](https://formspree.io/html). kindly fix this.
- The form method should be POST. kindly add the method so one can make a post request to the form when they submit the form.